brave/brave-core
# Replaced upstream clipboard icons with ours

fix https://github.com/brave/brave-browser/issues/39323

Allowed
<img width="131" alt="Screenshot 2024-06-28 at 11 42 04 AM" src="https://github.com/brave/brave-core/assets/6786187/88725bd5-2d8b-48dd-82f6-b11a7f650d86">

Blocked
<img width="119" alt="Screenshot 2024-06-28 at 11 42 19 AM" src="https://github.com/brave/brave-core/assets/6786187/b70370db-7002-4025-92d9-7a2eb9c60ede">



Resolves



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Load https://googlechrome.github.io/samples/async-clipboard/
2. Click paste button of the above web page
3. Click block or allow from permission request bubble
4. Check updated clipboard icon
